docs/Deploying-Pega-on-EKS.md

```diff
@@ -0,0 +1,578 @@
+  name: eks-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+-kind: ServiceAccount
+ name: eks-admin
+ namespace: kube-system
+```
+
+6. Apply the service account and cluster role binding to your cluster.
+
+```yaml
+$ kubectl apply -f eks-admin-service-account.yaml
+service account/eks-admin created
+```
+
+7. To generate an authentication token for the eks-admin service account required to connect to the dashboard of your administrative service account, enter:
+
+`kubectl -n kube-system describe secret $(kubectl -n kube-system get secret | grep eks-admin | awk '{print $1}')`
+
+8. Copy the <authentication_token> value to connect to the Kubernetes dashboard.
+
+9. To start the proxy server for the Kubernetes dashboard, enter:
+
+    `$ kubectl proxy`
+
+10. To access the Dashboard UI, open a web browser and navigate to the following URL:
+
+    `http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/`
+
+11. In the **Kubernetes Dashboard** sign in window, choose to use a cluster Kubeconfig token: select **Token** and paste the <authentication_token> value that you just generated into the **Enter token** area.
+
+12. Click **SIGN IN**.
+
+    You can now view your EKS cluster details using the Kubernetes dashboard. After you install Pega software, you can use this dashboard to review the status of all of the related Kubernetes objects used in your deployment. Without a deployment, only EKS resources display. The dashboard does not display your EKS cluster name or your resource name, which is expected behavior.
+
+    To continue using the Kubernetes dashboard to see the progress of your deployment, keep this PowerShell or Linux shell open.
+
+At this point, you must create a database instance into which you must install Pega Platform.
+
+### Creating a database resource
+
+Amazon EKS deployments require that you install Pega Platform software in an Amazon RDS instance that contains a PostgreSQL database. In general, it is recommended, but not required to name this database `pega`; the idea is to ensure that your organization recognizes the AWS database resource in which you install Pega Platform rules and the data your Pega deployment manages.
+
+During the configuration screens with which you create an RDS instance for your EKS deployment, you can create a database and name it `pega` and add a password during the DB instance provisioning in the **Additional configuration** section. After the DB instance is created, this runbook section includes an optional section to review and optimize the database for any specific organization requirements before you install and deploy Pega using this database.
+
+#### Creating an Amazon RDS instance
+
+Create a database that is available to your EKS cluster. This example creates an Amazon RDS instance in AWS; however, you can create or use any database resource that is available to your EKS cluster. 
```

Comment:
Is this suggesting that they could use something like an on-premise database?  This would be extremely slow